flink-tests/src/test/java/org/apache/flink/test/accumulators/AccumulatorLiveITCase.java

```diff
@@ -167,6 +172,9 @@ private static void submitJobAndVerifyResults(JobGraph jobGraph) throws Exceptio
 			NotifyingMapper.shutdownLatch.trigger();
 		} finally {
 			NotifyingMapper.shutdownLatch.trigger();
+
+			// wait for the completion of the job
+			clusterClient.requestJobResult(jobGraph.getJobID()).get();
```

Comment:
instead of introducing a new client interface _this late in the porting_ we could instead poll until the job is in a `FINISHED` state.

Reply:
another pattern we could employ is do a blocking submit in another thread and wait for that one to terminate.

Reply:
It is true that this introduces a new pattern, but I think it reflects more how the future `ClusterClient` abstraction will work. The `RestClusterClient` already has these methods and uses them to implement the `ClusterClient#submitJob` call.

I think it is not a big problem that some of the tests use polling and this one waits on the future. They can be refactored once we introduce the new `ClusterClient` interface. Moreover, this tests runs explicitly only with Flip-6.

Reply:
@zentol would you object merging the PR like it is? If so, then I can also apply one of the patterns you've proposed.

Reply:
I would really prefer if we would stick to existing patterns for now, otherwise we end up with a barely-started interface in 1.5 that is used for a single test :/

Reply:
Alright, will rework the test.